dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/get-utilisation-report-reconciliation-details.ts

```diff
@@ -34,7 +34,7 @@ export const getUtilisationReportReconciliationDetails = async (
 
   const feeRecordPaymentEntityGroups = getFeeRecordPaymentEntityGroupsFromFeeRecordEntities(feeRecords);
 
-  const feeRecordPaymentGroups = mapFeeRecordPaymentEntityGroupsToFeeRecordPaymentGroups(
+  const feeRecordPaymentGroups = await mapFeeRecordPaymentEntityGroupsToFeeRecordPaymentGroups(
```

Comment:
Should the unit test for this file be updated? It looks like it's assert empty arrays for `feeRecordPaymentGroups` and `keyingSheet`?

Reply:
No - we test the in-depth implementation of the mapping function in the unit tests for that mapping function, so we don't repeat it here to keep it DRY

Reply:
Where is the `feeRecordPaymentGroups` or `keyingSheet` population being unit tested? I cannot see in the test for this file/function. It's currently doing this:

```ts
expect(mappedReport).toEqual(
  // ...
  feeRecordPaymentGroups: [],
  keyingSheet: [],
);
```

But in the actual function, both of these properties are await/async. It sounds like some stubbing or something is required in order to fully test the function.

Reply:
The way to test that would simply be to test that those functions are being called. We can do that, but it feels a bit redundant to do so (not to mention that returning empty arrays is technically the correct implementation when there are no fee records)

Reply:
Helpers have been mocked